plugins/azure-devops-backend/src/api/AzureDevOpsApi.ts

```diff
@@ -47,14 +48,33 @@ import { TeamMember as AdoTeamMember } from 'azure-devops-node-api/interfaces/co
 import { Logger } from 'winston';
 import { PolicyEvaluationRecord } from 'azure-devops-node-api/interfaces/PolicyInterfaces';
 import { WebApi } from 'azure-devops-node-api';
-import { WebApiTeam } from 'azure-devops-node-api/interfaces/CoreInterfaces';
+import {
+  TeamProjectReference,
+  WebApiTeam,
+} from 'azure-devops-node-api/interfaces/CoreInterfaces';
 
 export class AzureDevOpsApi {
   public constructor(
     private readonly logger: Logger,
     private readonly webApi: WebApi,
   ) {}
 
+  public async getProjects(): Promise<Project[]> {
+    this.logger?.debug(`Getting all projects.`);
```

Comment:
Let's drop this, it's essentially gonna be logged through the request anyway

Reply:
Removed the log statement.